### PR TITLE
Fix some minor issues in docs

### PR DESCRIPTION
**Pull request by xerbalind**

### CODE REVIEW

File: mkdocs/docs/HPC/FAQ.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 {% if site == gent %}
 
-New users should consult the [Introduction to HPC](https://www.ugent.be/hpc/en/training/2023/introhpcugent)
-and the [HPC user manual](../index) to get started. This web page
+New users should consult the [Introduction to HPC](../introduction)
+and the [HPC user manual](../) to get started. This web page
 is a great resource for troubleshooting and looking up specifics.
```
Comment (xerbalind) [inline mkdocs/docs/HPC/FAQ.md:7]: is this still relevant?

File: mkdocs/docs/HPC/FAQ.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 {% if site == gent %}
 
-New users should consult the [Introduction to HPC](https://www.ugent.be/hpc/en/training/2023/introhpcugent)
-and the [HPC user manual](../index) to get started. This web page
+New users should consult the [Introduction to HPC](../introduction)
```
Comment (boegel) [inline mkdocs/docs/HPC/FAQ.md:5]:
```suggestion
New users should consult the [Introduction to HPC](introduction.md)

File: mkdocs/docs/HPC/FAQ.md (hunk 1 above)
Comment (boegel) [reply mkdocs/docs/HPC/FAQ.md:7]: No, not really, I would reword it to:

```
New users should consult the Introduction to HPC 
to get started, which is a great resource for learning the basics, troubleshooting, and looking up specifics.
```

File: mkdocs/docs/HPC/index.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Welcome to the {{ hpcinfra }} documentation!
 
 Use the menu on the left to navigate, or use the search box on the top right.
 
-If you still have any questions, you can [contact the {{ hpcteam }}]({{ hpc_support_url }}).
+This is the documentation from the point of view of a **{{ OS }}** user.
+switch to different operating system, click [here](../#force_select_OS).
```
Comment (boegel) [inline mkdocs/docs/HPC/index.md:8]:
```suggestion
To switch to different operating system (Windows, macOS, Linux), click [here](../#force_select_OS).

File: mkdocs/docs/HPC/index.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Welcome to the {{ hpcinfra }} documentation!
 
 Use the menu on the left to navigate, or use the search box on the top right.
 
-If you still have any questions, you can [contact the {{ hpcteam }}]({{ hpc_support_url }}).
+This is the documentation from the point of view of a **{{ OS }}** user.
```
Comment (boegel) [inline mkdocs/docs/HPC/index.md:7]:
```suggestion
The current view of the documentation is intended for people using **{{ OS }}* as operating system.

File: mkdocs/docs/HPC/program_examples.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
 {% set exampledir="examples/Program-examples" %}
 # Program examples { #ch:program-examples}
 
+If you have **not done already** copy our examples to your home directory:
```
Comment (boegel) [inline mkdocs/docs/HPC/program_examples.md:4]:
```suggestion
If you have **not done so already** copy our examples to your home directory by running the following command:
```

Maybe also add a note to explain what `~` is (or point to a section where that's explained)

File: mkdocs/docs/HPC/running_batch_jobs.md [hunk 6]
```diff
@@ -774,7 +774,7 @@ will start on `{{ othercluster }}` instead of the default `{{ defaultcluster }}`
 Note: the `{{ othercluster }}` modules may not work directly on the login nodes, because the
 login nodes do not have the same architecture as the `{{ othercluster }}` cluster, they have
 the same architecture as the `{{ defaultcluster }}` cluster however, so this is why by default
-software works on the login nodes. See [the section on Running software that is incompatible with host](../intro-HPC/troubleshooting/#running-software-that-is-incompatible-with-host) for why this is and how to fix
+software works on the login nodes. See [the section on Running software that is incompatible with host](../troubleshooting/#running-software-that-is-incompatible-with-host) for why this is and how to fix
```
Comment (boegel) [inline mkdocs/docs/HPC/running_batch_jobs.md:777]:
```suggestion
software works on the login nodes. See [the section on Running software that is incompatible with host](troubleshooting.md#running-software-that-is-incompatible-with-host) for why this is and how to fix

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
